app/controllers/solutions_controller.rb

```diff
@@ -1,2 +1,14 @@
 class SolutionsController < ApplicationController
-end
+ def create
+    @question = Question.find(params[:question_detail])
```

Comment:
この書き方で取得できますか？次の形式だと思いますよ。

```suggestion
    @question = Question.find(params[:question_id])
```